centos.org/ansible/provision.yml

```diff
@@ -41,7 +41,7 @@
       copy:
         content: |
           {% for host, value in cico_data.ansible_module_results.hosts.items() %}
-          {{ value.hostname }} ansible_fqdn={{ value.hostname }} ansible_user=root ansible_ssh_private_key_file=~/.ssh/id_rsa ansible_host={{ value.ip_address }}
+          {{ value.hostname }} ansible_fqdn={{ value.hostname }} ansible_user=root ansible_ssh_private_key_file=/duffy-ssh-key/ssh-privatekey ansible_host={{ value.ip_address }}
```

Comment:
If we don't specify the private key file, does it respect the SSH config that's already in place?

Reply:
Yeah, it probably does.